app/assets/javascripts/AngularConfig.js.coffee

```diff
@@ -706,7 +708,7 @@ angular.module('dahlia.controllers',['ngSanitize', 'angular-carousel', 'ngFileUp
     $translateProvider
       .preferredLanguage('en')
       .fallbackLanguage('en')
-      .useSanitizeValueStrategy('escapeParameters')
+      .useSanitizeValueStrategy('sanitize')
```

Comment:
`escape` makes it turn <tag> into &lt;tag&gt;
`sanitize` gets rid of harmful tags (e.g. no <style> and <script> tags allowed) but makes it so that we can use some valid HTML inside of translations.